libs/BLEKit/include/internal/ServicesCharacteristics.h

```diff
@@ -37,10 +37,10 @@ namespace monitoring {
 	inline constexpr uint16_t uuid = 0x7779;
 
 	namespace characteristic {
-		inline constexpr uint16_t charging_status	 = 0x6783;
-		inline constexpr uint16_t screensaver_enable = 0x8369;
-		inline constexpr uint16_t soft_reboot		 = 0x8382;
-		inline constexpr uint16_t hard_reboot		 = 0x7282;
+		inline constexpr uint16_t charging_status		   = 0x6783;
+		inline constexpr UUID::LongUUIDBytes_t go_to_sleep = "Go to Sleep";
+		inline constexpr uint16_t soft_reboot			   = 0x8382;
+		inline constexpr uint16_t hard_reboot			   = 0x7282;
```

Comment:
I suggest moving long UUIDs down and also using the upper case snake case `NAME_OF_SERVICE` to avoid confusion between lower/upper case and letter such as `i I l L` (i I l L)

```suggestion
		inline constexpr uint16_t charging_status		   = 0x6783;
		inline constexpr uint16_t soft_reboot			   = 0x8382;
		inline constexpr uint16_t hard_reboot			   = 0x7282;
		
		inline constexpr UUID::LongUUIDBytes_t go_to_sleep = "GO_TO_SLEEP";
```